src/modules/css.js

```diff
@@ -0,0 +1,77 @@
+const getPropertyNameValue = (element, propertyName) => {
+    if (element.length === undefined) {                 
+        return window.getComputedStyle(element)[propertyName];
+    } else {
+        for (let item of element) {
+            let propertyNameValue = window.getComputedStyle(item)[propertyName];
+            if (propertyNameValue) {
+                return propertyNameValue;
+            }
```

Comment:
You can just `return propertyNameValue` without checking because your function will return undefined in other case( if property not exist too)